app/assets/javascripts/views/restrooms/restrooms.js.coffee

```diff
@@ -3,15 +3,20 @@
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
 $ ->
-  $('#ada_filter').click -> 
+  $('#ada_filter').click ->
     if $(this).hasClass("active")
       $('.listItem.not_accessible').show()
-    else 
+    else
       $('.listItem.not_accessible').hide()
-    
-  $('#unisex_filter').click -> 
+
+  $('#unisex_filter').click ->
     if $(this).hasClass("active")
       $('.listItem.not_unisex').show()
-    else 
+    else
       $('.listItem.not_unisex').hide()
 
+  $('#changing_table_filter').click ->
```

Comment:
I don't really think there is a rails convention that would help here. These technically are three different filters. There may be a gem that will help with this, but since we are doing this filtering in javascript -- I don't know if it would really help without a refactor here.  I'm fine with this for now.